README.md

```diff
@@ -28,12 +28,48 @@ When the reader has completed this Code Pattern, they will understand how to:
 
-## Prerequisites:
+## Featured technologies
+
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be applied to disparate solution spaces to deliver disruptive technologies.
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
+
+## Prerequisites
 
 The following are prerequisites to start developing this application:
 
 * Xcode
 * Turi Create (5.0 Beta 2)
 * Cocoapods
 * Python 2.7 & PIP
+* Flask
 * Jupyter
+* ngrok
+
+# Steps
+
+(NOTE: I still need to add screenshots/pictures to all the steps!)
+
+1. [Download and organize the latest MovieLens dataset](#1-download-and-organize-the-latest-movielens-dataset)
+2. [Train the Turi Create Recommender model](#2-train-the-turi-create-recommender-model)
+3. [Run the backend](#3-run-the-backend)
+4. [Deploy the iOS app](#4-deploy-the-ios-app)
+
+### 1. Download and organize the latest MovieLens dataset
+
+There's a script available in the root directory of this repository, called `setup.sh`. Run this script - it'll download the movielens data, and organize it, for you.
+
+### 2. Train the Turi Create Recommender model
+
+In order for the recommendations to work, you need a trained model. Turi Create is a Python package that will train this model for you.
+
+Go ahead and start your Jupyter Notebooks. In the `local_model_training` folder, open the `TrainRecommendationModel.ipynb` notebook. Run the notebook, and once it's done executing, you're going to have a folder called `movie_rec`, which contains the trained Turi Create model. Unlike other machine learning libraries, Turi Create does not use files for models - it uses folders.
+
+### 4. Run the backend
+
+Turi Create supports exporting recommender models to CoreML, so they can run on-device, and there's no need for a backend. However, it only works with iOS 12 and above (which is currently in Beta), and there's an [issue with linking the custom model framework](https://github.com/apple/turicreate/issues/799). Therefore, this code pattern will use a backend that takes requests from the iOS Application, and runs the users' preferences through the turi create model, to return predictions as to what the user would like to watch next.
+
+First, copy the `movie_rec` folder (your trained model) from `local_model_training`, and paste it into the `serverside_prediction_api` folder. Once that's done, run the following command from a terminal window within that folder: `FLASK_APP=backend.py flask run`. This will run the `backend.py` file, which is the flask application.
+
+### 5. Deploy the iOS app
+
+Finally, you can run the front-end of the application. Go ahead into the `iOS_frontend/MovieRecommender` folder, and open the `xcworkspace` file. Once you're there, run the application by hitting `Cmd+R` or clicking the run button beside the window controls on the top left of Xcode.
```

Comment:
Again, just to help it read more smoothly: "Go ahead into the" -> "Navigate to the"

You could also use <kbd>CMD</kbd> + <kbd>R</kbd> instead of the literal.

    <kbd>CMD</kbd> + <kbd>R</kbd>